### PR TITLE
developed a birthday card using HTML and CSS

### PR DESCRIPTION
**Pull request by Vimal2023**
Responsive 

Birthday Card - HTML & CSS. - Vimal Anand

Screenshot (204)
Screenshot (205)

### CODE REVIEW

**Review by Rishikavishnoi — COMMENTED** [review]
@Vimal2023 Have you directly raised the PR ..?? First, create an issue  that will be assigned to you and then raise the PR

**Review by LiQuiD-404 — CHANGES_REQUESTED** [review]
@Vimal2023 Kindly add a Readme.md file

**Comment by Anubhavgupta14** [comment]
Mention the Issue no. for this PR, if not created the issue then first create an issue then wait for the approval then you have to raise new PR for that issue.

**Pull request closed without merging**
